floyd/cli/data_upload_utils.py

```diff
@@ -76,9 +85,11 @@ def complete_upload(data_config):
     floyd_logger.debug("Getting fresh upload credentials")
     creds = DataClient().new_tus_credentials(data_id)
 
-    floyd_logger.info("Uploading compressed data. Total upload size: {}".format(sizeof_fmt(file_size)))
+    floyd_logger.info("Uploading compressed data. Total upload size: %s",
```

Comment:
Yes, I am planning to change all logging statements to this format. This should be the preferred way for doing logging in python. If we use format, the actual log string will get constructed at runtime regardless of the logging level. In the past, I have experienced 2X-3X performance improvement with this change in a production system. It was creating a lot of strings at runtime with `logger.debug` even with logger level set to `warning` :P